noir_stdlib/src/hash/poseidon2.nr

```diff
@@ -1,7 +1,7 @@
 use crate::hash::Hasher;
 use crate::default::Default;
 
-global RATE: u32 = 3;
+comptime global RATE: u32 = 3;
```

Comment:
Needed to mark this as global to be used in the comptime hashing code.

It's a bit awkward to do this automatically since it means lazily evaluating globals in the interpreter which aren't marked comptime. This could change program behavior if the evaluating the global has side-effects. So I opted to just mark this as comptime for now.